src/nvim/testdir/test_taglist.vim

```diff
@@ -76,6 +76,8 @@ func Test_tagsfile_without_trailing_newline()
 endfunc
 
 func Test_tagfiles()
+  " Nvim: defaults to "./tags;,tags", which might cause false positives.
+  set tags=./tags,tags
```

Comment:
Can you move this in `src/nvim/testdir/setup.vim`?

Reply:
Ok, done.